src/main.cpp

```diff
@@ -191,6 +192,15 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // JSON-RPC bind address ------------------------------------------------
+        if ( GetStringArgument ( argc, argv, i, "--jsonrpcbind", "--jsonrpcbind", strArgument ) )
```

Comment:
Hmm, thinking of `--serverbindip` this should be `--jsonrpcbindip` for consistency, I guess?
(Other places would need to be adapted as well, of course)